homeassistant/components/devolo_home_network/__init__.py

```diff
@@ -127,6 +128,19 @@ async def async_update_led_status() -> bool:
         except DeviceUnavailable as err:
             raise UpdateFailed(err) from err
 
+    async def async_update_uptime() -> int:
```

Comment:
Slightly pedantic but maybe this should be called `async_update_last_restart` to match the other update methods that are named after the associated sensor type.

Reply:
That's indeed a good idea.